src/WebJobs.Extensions.DurableTask/Listener/OutOfProcOrchestrationShim.cs

```diff
@@ -196,6 +196,12 @@ private class AsyncAction
 
             [JsonProperty("httpRequest")]
             internal DurableHttpRequest HttpRequest { get; set; }
+
+            [JsonProperty("entity")]
+            internal string EntitySchedulerId { get; set; }
```

Comment:
This field is actually redundant, since the entity scheduler id is actually equivalent to the orchestration instance id. 

Technically I believe that `EntityOperation` is also an event behind the scenes, though the that value is not actually the event that gets sent to the orchestration (all of these are translated to the "op" event with the operation name embedded in the body). It might be best to treat it as it's own field here.

Reply:
Using separate fields seems fine to me. If we wanted to optimize a little more we could share the field and use a name for that field that reflects both purposes (e.g. `'EventOrOperationName`). Not likely that it would make much of a difference.